openmc/nuclide.py

```diff
@@ -98,7 +99,14 @@ def scattering(self):
     @name.setter
     def name(self, name):
         check_type('name', name, basestring)
-        self._name = name
+
+        if '-' in name:
+            new_name = name.replace('-', '')
+            msg = 'OpenMC nuclide names follow the GND standard. Nuclide ' \
```

Comment:
Since they don't have a standard yet ... "follow the proposed GND naming convention" or something like that.